### ISSUE TITLE
Cannot read property 'backend' of undefined when running the face-landmarks-detection in a comlink web worker 

### ISSUE THREAD

**Issue by driescroons** [tensorflow/tfjs#4841]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow.js): no
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows 10
- TensorFlow.js installed from (npm or script link): 3.3.0
- Browser version: Chrome 89.0.4389.90

**Describe the current behavior**
I ran into an issue building a snapchat filter clone app thing. I've setup an example project of the same issue I had. I'd like to run the model in a web worker. When I move my landmark face detection code to a comlink web worker, I get following issue:
![image](https://user-images.githubusercontent.com/27679518/111815707-482ca480-88dc-11eb-9ded-3b53617a5659.png)

**Describe the expected behavior**
It should work as if it ran in the main thread.

**Standalone code to reproduce the issue**
codesandbox: https://codesandbox.io/s/github/driescroons/tensorflow-comlink-issue (gives an error on the loader, so I suggest you use the gh link)
github link: https://github.com/driescroons/tensorflow-comlink-issue

In following diff you can see I just move the code from the already working app to the comlink webworker:
https://github.com/driescroons/tensorflow-comlink-issue/commit/1c239d4a14406c825e90e17e4e3f39a2a3c316ec

Let me know if you need anything else!

**Comment by rthadur** [comment]
Thanks for the reproduction code , I could not run the code as I was getting below error ,

![image](https://user-images.githubusercontent.com/43972606/112201899-fd59a800-8bcd-11eb-99ad-b6bb3870c351.png)

**Comment by driescroons** [comment]
as mentioned, you need to check the gh repo. The comlink-loader does not seem to work within csb.

**Comment by driescroons** [comment]
any update on this? Let me know if you need any more examples / help.

**Comment by jinjingforever** [comment]
Sorry Dries, I have been busy with other projects, but I will take a look at this issue some time this week. Thanks for providing the sample project.

**Comment by driescroons** [comment]
Thanks for the reply @jinjingforever. I've tried what you suggested and got it to work. Seems like rendering the video to an offscreen canvas and passing the imageData to my webworker is a bit slower than running everything in the main thread though. I should do some benchmarking.

For anyone else, here are the changes I've made:
https://github.com/driescroons/tensorflow-comlink-issue/compare/feat/imagedata
